app/services/backfill_notifications.rb

```diff
@@ -0,0 +1,15 @@
+class BackfillNotifications
+  def self.call
+    Communication.find_each do |communication|
+      notification = communication.build_notification
+      notification.message = "sms.notifications.#{communication.communication_type}"
```

Comment:
I think this needs to be changed since you changed the locales key back to "appointment_reminders".

Reply:
oh right, fixed